src/matchers.test.ts

```diff
@@ -131,3 +139,147 @@ describe('isAllOf', () => {
     ).toBe(false)
   })
 })
+
+describe('isPending', () => {
+  test('should return false for a regular action', () => {
+    const action = createAction<string>('a')
```

Comment:
```suggestion
    const action = createAction<string>('action/type')('testPayload')
```
Otherwise you test the action creator, not the action.

Reply:
Fixed.